projects/packages/my-jetpack/src/class-products.php

```diff
@@ -107,6 +108,7 @@ class Products {
 		self::STATUS_NEEDS_PLAN,
 		self::STATUS_NEEDS_ACTIVATION,
 		self::STATUS_NEEDS_FIRST_SITE_CONNECTION,
+		self::STATUS_NEEDS_ATTENTION,
```

Comment:
Could this status belong in the `$warning_statuses` array as well?

Reply:
Yeah, probably so..  Thanks, will do. 👍